androidApp/src/main/java/com/mbta/tid/mbta_app/android/alertDetails/AlertDetailsPage.kt

```diff
@@ -93,7 +96,14 @@ fun AlertDetailsPage(
             horizontalArrangement = Arrangement.spacedBy(6.dp),
             verticalAlignment = Alignment.CenterVertically
         ) {
-            if (firstRoute != null) {
+            if (alert?.effect == Alert.Effect.ElevatorClosure) {
+                Image(
+                    painterResource(R.drawable.elevator_alert_monochrome),
+                    "",
```

Comment:
`Image` accepts `contentDescription = null`, which I think is clearer than `contentDescription = ""`.